base/dcp_sharded.go

```diff
@@ -740,6 +743,10 @@ func (meh *sgMgrEventHandlers) OnUnregisterPIndex(pindex *cbgt.PIndex) {
 // This will trigger cbgt closing and then attempting to reconnect to the feed.
 func (meh *sgMgrEventHandlers) OnFeedError(srcType string, r cbgt.Feed, feedErr error) {
```

Comment:
nit, readabilty:

```suggestion
// OnFeedError returns an invalid srcType as the second argument
func (meh *sgMgrEventHandlers) OnFeedError(_ string, r cbgt.Feed, feedErr error) {
```

Reply:
I'm not completely convinced that this is an improvement to readability - going to leave this as-is for now.